llama_recipes/llama_finetuning.py

```diff
@@ -204,10 +204,12 @@ def main(**kwargs):
     
     else:
         kwargs['r'] = kwargs['lora_rank'] # can't pass --r to the script, torchrun won't have it
+        kwargs['target_modules'] = list(train_config.target_modules)
```

Comment:
ignorant question - I assume this framework automatically splits the comma-separated list of modules?